### ISSUE TITLE
restic version tries resolve password

### ISSUE THREAD

**Issue by minusf** [restic/restic#1632]
## Output of `restic version`

:}  this is exactly the problem.

```
$ echo $RESTIC_PASSWORD_FILE
/path/to/password_file
$ restic version
Resolving password failed: Fatal: /path/to/password_file does not exist
```

```
restic 0.8.2
compiled with go1.10 on darwin/amd64
```

## How did you run restic exactly?

$ restic version

## What backend/server/service did you use to store the repository?

n/a

## Expected behavior

show the version number

## Actual behavior

trying to resolve password

## Do you have an idea how to solve the issue?

i think it's fine to show the version number without resolving the password :}
actually i see that even `restic help version` doesn't work if the pwd file cannot be resolved.

## Did restic help you or made you happy in any way?

don't be so hard on yourself, restic :}

**Comment by minusf** [comment]
(btw the user case is, that the password file is on an encrypted external drive, where the backup is also written.  the disk is not always attached, hence the file is not available unless i am backing up or restoring.)

**Comment by fd0** [comment]
Thanks for the report. The reason here is that restic tries to read the file when the parameters are parsed. This could be changed though.

**Comment by ebastos** [comment]
Took a stab at it.
Please, let me know your thoughts.

**Comment by fd0** [comment]
Resolved in #1662, thanks @ebastos!
